python/cuml/test/test_kernel_density.py

```diff
@@ -0,0 +1,140 @@
+#
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from cuml.neighbors import KernelDensity, VALID_KERNELS, logsumexp_kernel
+from sklearn.metrics import pairwise_distances as skl_pairwise_distances
+from sklearn.neighbors._ball_tree import kernel_norm
+import numpy as np
+from hypothesis import given, settings, assume, strategies as st
+from hypothesis.extra.numpy import arrays
+import pytest
+from sklearn.model_selection import GridSearchCV
+
+
+# not in log probability space
+def compute_kernel_naive(Y, X, kernel, metric, h, sample_weights):
```

Comment:
Not very important but it would be nice to always use `sample_weight` in this test file, without the `s` at the end to stay consistent with the spelling of the KDE code.